src/blockdescription.jl

```diff
@@ -55,4 +55,4 @@ function Base.show(io::IO, ::MIME"text/plain", b::BlockDescription)
     end
 end
 
-Base.show(io::IO, b::BlockDescription) = show(io, MIME"text/plain"(), b)
+Base.show(io::IO, b::BlockDescription) = show(io, MIME("text/plain"), b)
```

Comment:
This is the same, isn't it? Is there a reason to prefer one over the other?

Reply:
It's exactly the same. I just decided that the former looks really weird and I don't like it....